scripts/setup.sh

```
@@ -0,0 +1,43 @@
+
+# Check if pip is installed
+if ! command -v pip3 &> /dev/null; then
+    echo "❌ pip3 is not installed. Please install pip3."
+    exit 1
+fi
+
+echo "✅ pip3 found: $(pip3 --version)"
+
+# Install Python dependencies
+echo "📦 Installing Python dependencies..."
+cd "$(dirname "$0")"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)